cloudbuild-stackdriver.yaml

```diff
@@ -0,0 +1,11 @@
+steps:
+- name: 'debian:jessie'
+  dir: spinnaker-monitoring-daemon
+  entrypoint: "bash"
+  args: ["-c", "echo 'google-api-python-client' >> requirements.txt; echo 'oauth2client' >> requirements.txt"]
+- name: 'gcr.io/cloud-builders/docker'
+  dir: spinnaker-monitoring-daemon
+  args: ["build", "-t", "gcr.io/$PROJECT_ID/$REPO_NAME:$COMMIT_SHA", "-t", "gcr.io/$PROJECT_ID/$REPO_NAME:latest", "-f", "Dockerfile", "."]
```

Comment:
@ttomsu should I make sure these image names include the monitoring solution type? Or is that handled in the trigger setup screen on GCB?